README.md

```diff
@@ -1,9 +1,6 @@
 # ACE: AI2 Climate Emulator
 This repo contains the inference code accompanying "ACE: A fast, skillful learned global atmospheric model for climate prediction" ([arxiv:2310.02074](https://arxiv.org/abs/2310.02074)).
 
-## DISCLAIMER
-This is rapidly changing research software. We make no guarantees of maintaining backwards compatibility.
```

Comment:
Do we mean to remove this disclaimer?

Reply:
Yes, this was intentional. There is a similar warning in the docs.